### DISCUSSION TITLE
Report | Week 13

### DISCUSSION THREAD

**Discussion by tech-n1c** [SE-TINF22B6/time2shine#78] [Show and tell]
Hello everyone,

last week, we made quite some changes to our website. The landing page now shows users what features we have to offer and also provides quick links to the games overview and leaderboards.
![image](https://github.com/SE-TINF22B6/time2shine/assets/59262249/961f3b58-d89a-4050-9069-0228e578db6f)

---

The game overview page has also been reworked and now shows all the games we offer in a convenient style:
![image](https://github.com/SE-TINF22B6/time2shine/assets/59262249/2c842e74-ed8d-4649-9704-70997a691de3)
As you can see, we added another game, called "Röhrig Clicker", which is about a crazy professor who needs some help grading his student's exams. This can be done by clicking him as often as possible. The player is also able the parallelize the Röhrigs by buying new instances. Check out our new game and tell us what you think about it!

---

![image](https://github.com/SE-TINF22B6/time2shine/assets/59262249/1a5dcfeb-b191-4bc6-a7fa-9ebfc52a895f)
After the users submit their score, they can check if they made it to our leaderboard, where the highscores of all games are shown.
---

We're also working on another game that is similar to snake. Right now, we don't have any graphics for the game so it doesn't look as good but a lot of functionalities are already implemented, like building the game board, moving the snake(s) and splitting a snake. You'll need to wait a bit longer though to find out what the snake splitting is about.

**Comment by MaarcS** [comment]
Dear time2shine team,

Your redesign of your website looks very professional, and really fits to a product that is all about gaming.
I also like the idea of your new game "Röhrig Clicker". I wanted to check it out, but when I try to sign in with GitHub I get a "Server Error" message. Is this a known issue?

Your CookHub team

**Reply by tech-n1c** [reply]
Sorry for the late response. Yes, we are aware of that and it is caused by the CI/CD pipeline (but still thx for the report!). When the game gets deployed, we have to manually do some steps for the authentication, that is why it isnt always working.

**Comment by Shiny1708** [comment]
I really like the design of the UI. It gives a modern and comfortable feel when visiting your website. The Navigation also seems intuitive and easy to understand. I assume the pictures are just placeholders tight now, you could look into replacing them with screenshots from the actual games or logos that represent them
